.github/workflows/build-test

```diff
@@ -52,13 +100,8 @@ then
     python -m pip install --upgrade mypy
 fi
 for MODULE in $MODULES_TO_TEST; do
-    # pytket-iqm not supported with 3.7 or 3.8
-    if [[ "${MODULE}" != "pytket-iqm" || "${PYVER}" == "3.9" ]]; then
-        # FIXME see
-        # https://github.com/CQCL/pytket-extensions/issues/180 and
-        # https://github.com/microsoft/iqsharp/issues/512
-        if [[ "${MODULE}" != "pytket-qsharp" || "${PLAT}" != "Windows" || "${PYVER}" == "3.7" ]]
```

Comment:
Should we close the related issue now?

Reply:
As far as I know the issue is still open. We can build pytket-qsharp on Windows with 3.8 and 3.9, but we cannot test it (at least on the CI, though it seems to work in some local installations). With 3.10 we can't even build it. So unfortunately we can't test pytket-qsharp on Windows on the CI. However I didn't want to remove support altogether because there may be people using it, we can still test locally, and the issue may get fixed.

Reply:
Yes, you are right. I forgot that we had a problem with 3.8 and 3.9, too. We should keep it.